vars/MPLPipelineConfig.groovy

```diff
@@ -41,7 +43,14 @@ def call(body, Map defaults = [:], Map overrides = [:]) {
   if( body in Closure ) {
     body.resolveStrategy = Closure.DELEGATE_FIRST
     body.delegate = config
+
```

Comment:
OMG, i understood how it works..
Let`s add following comments:
Line 50: # Main goal of execute body() here is update pipeline "defaults" which will be merged with override later. Delegation allow us do it.

Reply:
Added comments to clarify